packages/p2p-media-loader-hlsjs/src/services/engine.ts

```diff
@@ -1,19 +1,108 @@
-import { HybridLoader, Math } from "p2p-media-loader-core";
+import Hls from "hls.js";
```

Comment:
This line includes all the Hls engine in the bundle. Can you please check?

I didn't find a good way to use only Hls.js types in the project though because of issues in the package type system.

Reply:
We can do something like this

import type Hls from "hls.js";
import type { HlsConfig } from "hls.js";

Hls class type is exported by default.
But we also should use plain strings instead of Hls.Events.DESTROYING enum.

Another obvious solution is to write necessary type manually due.
I had this class imported because previously I used it to create instances exactly in this package.